nightfall-optimist/src/services/debug-counters.mjs

```diff
@@ -2,11 +2,11 @@ const debugCounters = {
   // CheckBlock returns a failure
   nBlockInvalid: 0,
   // Proposer WS is closed
-  proposerWsClosed: 0,
-  // Proposer WS is clossed and coulnd't be opened
-  proposerWsFailed: 0,
-  // Optimist couldnt proposed blocks because of WS
-  proposerBlockNotSent: 0,
+  // proposerWsClosed: 0,
```

Comment:
We can kick these commented lines out :)

Reply:
ec704738